host/4/bookworm/dotnet-isolated/dotnet9-appservice.Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Build the runtime from source
```

Comment:
Do we expect to add bookworm versions of other .net isolated versions (DNI10 or DNI8). If so, should we consider moving the files under "dotnet-isolated9" directory? I see we did that for the bullseye images.

Reply:
We could do that in the future. We are moving from this repo tho so lets not worry about it